export function handleError(error: Error) {
  console.error(error);
  process.exit(1);
}
